dev-vcs/gitolite/gitolite-3.6.11-r1.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+[[ ${PV} == *9999 ]] && SCM="git-2"
+EGIT_REPO_URI="https://github.com/sitaramc/${PN}.git"
+EGIT_MASTER=master
+
+inherit perl-module user versionator ${SCM}
+
+DESCRIPTION="Highly flexible server for git directory version tracker"
+HOMEPAGE="https://github.com/sitaramc/gitolite"
+if [[ ${PV} != *9999 ]]; then
+	SRC_URI="https://github.com/sitaramc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~x86"
+else
+	SRC_URI=""
+	KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
I think unstable keywords in live ebuild is incorrect. We can also remove the `SRC_URI=""`